examples/prometheus/node-exporter.yaml

```diff
@@ -0,0 +1,177 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: node-exporter
+  annotations:
+    "openshift.io/display-name": "Node Exporter"
+    description: |
+      Prometheus Node Exporter - expose node-level metrics for consumption by Prometheus.
+    iconClass: icon-cogs
+    tags: "monitoring,prometheus,time-series,node-exporter"
+parameters:
+- description: The namespace to instantiate node-exporter under. Defaults to 'kube-system'.
+  name: NAMESPACE
+  value: kube-system
+- description: The location of the proxy image
+  name: IMAGE_PROXY
+  value: registry.svc.ci.openshift.org/ci/oauth-proxy:latest
+- description: The location of the node-exporter image
+  name: IMAGE_NODE_EXPORTER
+  value: quay.io/prometheus/node-exporter:latest
+- description: The session secret for the proxy
+  name: SESSION_SECRET
+  generate: expression
+  from: "[a-zA-Z0-9]{43}"
+objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: node-exporter
+    namespace: "${NAMESPACE}"
+    annotations:
+      serviceaccounts.openshift.io/oauth-redirectreference.primary: '{"kind":"OAuthRedirectReference","apiVersion":"v1","reference":{"kind":"Route","name":"node-exporter"}}'
+# Allow the node-exporter to access data and mounts from the host,
+# so that it can expose metrics about them.
+- apiVersion: v1
+  kind: SecurityContextConstraints
+  metadata:
+    name: node-exporter
+  allowHostDirVolumePlugin: true
+  allowHostNetwork: true
+  allowHostPID: true
+  allowHostPorts: true
+  runAsUser:
+    type: RunAsAny
+  seLinuxContext:
+    type: RunAsAny
+  users:
+  - system:serviceaccount:${NAMESPACE}:node-exporter
+  volumes:
+  - hostPath
+  - secret
```

Comment:
:+1: I figured you were back already since it's 3:30 am on the east coast.  I hope you're at least across the ocean already and not just up really late. :smile: Safe travels!